src/main/java/uk/gov/hmcts/sptribs/caseworker/event/RespondentContactParties.java

```diff
@@ -134,26 +133,31 @@ public SubmittedCallbackResponse partiesContacted(CaseDetails<CaseData, State> d
                 .confirmationHeader(format("# Contact Parties notification failed %n## Please resend the notification"))
                 .build();
         }
+
         return SubmittedCallbackResponse.builder()
             .confirmationHeader(format("# Message sent %n## %s",
-                MessageUtil.generateSimpleMessage(details.getData().getContactParties())
+                MessageUtil.generateSimpleMessage(data.getContactParties())
             ))
             .build();
-
     }
 
     private void sendContactPartiesNotification(CaseDetails<CaseData, State> details, CaseData data, String caseNumber) {
-        if (!CollectionUtils.isEmpty(data.getContactParties().getSubjectContactParties())) {
-            contactPartiesNotification.sendToSubject(details.getData(), caseNumber);
-        }
-        if (!CollectionUtils.isEmpty(data.getContactParties().getRepresentativeContactParties())) {
-            contactPartiesNotification.sendToRepresentative(details.getData(), caseNumber);
-        }
-        if (!CollectionUtils.isEmpty(data.getContactParties().getApplicantContactParties())) {
-            contactPartiesNotification.sendToApplicant(details.getData(), caseNumber);
-        }
-        if (!CollectionUtils.isEmpty(data.getContactParties().getTribunal())) {
-            contactPartiesNotification.sendToTribunal(details.getData(), caseNumber);
+        ContactParties contactParties = data.getContactParties();
+        CaseData caseData = details.getData();
```

Comment:
data is already passed as the second parameter no?

Reply:
yeah, I too thought the same. But, it was used from details before, so don't want to break anything as part of the cleanup. So, I used the same.